api/translate_names/translate_names.py

```diff
@@ -0,0 +1,36 @@
+from api import util
+
+
+def translate_names_main(name_type, name, name_type2, major=None, minor=None):
+    if name_type not in ["distgit", "dist-git"]:
+        return {
+                   "status": "error",
+                   "payload": f"Sorry, don't know how to look up a {name_type} yet."
+               }, 400
+    if name_type2 not in ["brew-image", "brew-component"]:
+        return {
+                   "status": "error",
+                   "payload": f"Sorry, don't know how to translate to a {name_type2} yet."
+               }, 400
+
+    query_name = {
+        "brew-image": "image_name",
+        "brew-component": "component",
+    }[name_type2]
+    major_minor = f"{major}.{minor}" if major and minor else "4.5"
```

Comment:
whoah, 4.5??

hmm, not sure i want a default for this at all. just make major/minor required

Reply:
I replaced it as the GA version